e2e/config/services_shannon.yaml

```diff
@@ -100,6 +101,7 @@ services:
     alias: "base-sepolia-testnetnet"
     service_type: "evm"
     archival: true
+    websockets: true
```

Comment:
Can you set `websockets: false` on the other ones?

It'll make it easier to toggle/read/understand for readers.